spec/ssl-test_spec.rb

```diff
@@ -8,7 +10,16 @@
 describe SSLTest do
   before { SSLTest.flush_cache }
 
-  describe '.test' do
+  let(:proxy_thread) { nil }
+
+
+  after(:each) do
+    if proxy_thread
+      proxy_thread.kill
+    end
+  end
```

Comment:
I would probably shorten this as:
```suggestion
  after(:each) { proxy_thread&.kill }
```

Reply:
Great, thank you :)